opensrp-anc/src/main/assets/config/profile-overview.yml

```diff
@@ -37,10 +37,6 @@ fields:
 ---
 sub_group: obstetric_history
 fields:
-  - template: "{{profile_overview.overview_of_pregnancy.obstetric_history.gravida}}: {gravida}"
-    relevance: "gravida != ''"
-    isRedFont: "gravida >= 5"
-
```

Comment:
Is this also removed because it was on a wrong file?

Reply:
@junaidwarsivd  this was replaced with the new Gravida data dictionary.